docs/fernet.rst

```diff
@@ -53,6 +53,26 @@ has support for implementing key rotation via :class:`MultiFernet`.
             generated in *plaintext*, the time a message was created will
             therefore be visible to a possible attacker.
 
+    .. method:: encrypt_at_time(data, current_time)
```

Comment:
This needs a `.. versionadded:: 3.0` notation (as does `decrypt_at_time`). Otherwise this LGTM, thanks for working with us here.

Reply:
Thanks for the review again, done.